web_src/css/dashboard.css

```diff
@@ -21,6 +21,9 @@
   display: flex;
   align-items: center;
   justify-content: space-between;
+  line-height: 1.3; /* avoid cut-off letters because fomantic line-height and overflow hidden via ellipsis */
+  padding-top: 8.9px; /* adjust for increased line-height */
+  padding-bottom: 8.9px; /* adjust for increased line-height */
```

Comment:
I have seemed similar code before (and removed some of them)

IMO it's better to avoid such "fine-tuning", it seems pretty hacky.

Reply:
Does this `.1px` really have a visible impact? I'd say `9px` or even `8px` should be okay